views/homework/assignment.hbs

```diff
@@ -50,63 +50,73 @@
+                    <a id="extended-tab-link" class="tab-link active" href="#activetabid=extended"
+                        alt="Details der Aufgabe">Details</a>
                     {{#if private}}
-                        <a id="submission-tab-link" class="tab-link" href="#activetabid=submission" alt="Deine Abgabe">Abgabe</a>
+                        <a id="submission-tab-link" class="tab-link" href="#activetabid=submission"
+                            alt="Deine Abgabe">Abgabe</a>
                     {{else}}
-                        {{#ifvalue teacherId value=currentUser._id}}
-                            <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions" alt="alle Abgaben">Abgaben</a>
+                        {{#if isTeacher}}
+                            <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions"
+                                alt="alle Abgaben">Abgaben</a>
                         {{else}}
-                            {{#inArray currentUser._id courseId.substitutionIds}}
-                                <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions" alt="alle Abgaben">Abgaben</a>
-                            {{else}}
-                                <a id="submission-tab-link" class="tab-link" href="#activetabid=submission" alt="Deine Abgabe">Abgabe</a>
-                                <a id="feedback-tab-link" class="tab-link" href="#activetabid=feedback" alt="Bewertung deiner Abgabe durch den Lehrer">Feedback</a>
-                                {{#if ../publicSubmissions}}
-                                    <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions" alt="alle Abgaben">Abgaben</a>
-                                {{/if}}
-                            {{/inArray}}
-                        {{/ifvalue}}
+                            <a id="submission-tab-link" class="tab-link" href="#activetabid=submission"
+                                alt="Deine Abgabe">Abgabe</a>
+                            <a id="feedback-tab-link" class="tab-link" href="#activetabid=feedback"
+                                alt="Bewertung deiner Abgabe durch den Lehrer">Feedback</a>
+                            {{#if ../publicSubmissions}}
+                                <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions"
+                                    alt="alle Abgaben">Abgaben</a>
+                            {{/if}}
+                        {{/if}}
                     {{/if}}
                 </nav>
                 <div class="tabs">
-{{> 'homework/extended'}}
+                    {{> 'homework/extended'}}
                     {{#if private}}
-{{> 'homework/submission'}}
+                        {{> 'homework/submission'}}
                     {{else}}
-                        {{#ifvalue teacherId value=currentUser._id}}
-{{> 'homework/submissions'}}
+                        {{#if isTeacher}}

```

Comment:
Da teils auch Schüler in diese Ansicht kommen können finde ich `isTeacher` etwas verwirrend. Vllt. findest du noch eine passendere Bezeichnung? Vllt. sowas wie `canViewSubmissions`, `renderSubmissions`?